src/ext/issuance.js

```diff
@@ -39,56 +48,60 @@ function signReqCF(url) {
     sentTokens[reqUrl] = true;
 
     // Generate tokens and create a JSON request for signing
-    let tokens = GenerateNewTokens(TOKENS_PER_REQUEST);
+    const tokens = GenerateNewTokens(TOKENS_PER_REQUEST);
     const request = BuildIssueRequest(tokens);
 
     // Tag the URL of the new request to prevent an infinite loop (see above)
-    let newUrl = markSignUrl(reqUrl);
+    const newUrl = markSignUrl(reqUrl);
     // Construct info for xhr signing request
-    let xhrInfo = {newUrl: newUrl, requestBody: "blinded-tokens=" + request, tokens: tokens};
+    const xhrInfo = {newUrl: newUrl, requestBody: "blinded-tokens=" + request, tokens: tokens};
 
     return xhrInfo;
 }
 
+/**
+ * hCaptcha issuance request
+ * @param {URL} url
+ * @return {Object} XHR info for asynchronous token issuance
+ */
 function signReqHC(url) {
-    let reqUrl = url.href;
+    const reqUrl = url.href;
     const isIssuerUrl = ISSUE_ACTION_URLS
-        .map(issuerUrl => patternToRegExp(issuerUrl))
-        .some(re => reqUrl.match(re));
+        .map((issuerUrl) => patternToRegExp(issuerUrl))
+        .some((re) => reqUrl.match(re));
 
     if (!isIssuerUrl) {
         return null;
     }
 
     sentTokens[reqUrl] = true;
     // Generate tokens and create a JSON request for signing
-    let tokens = GenerateNewTokens(TOKENS_PER_REQUEST);
+    const tokens = GenerateNewTokens(TOKENS_PER_REQUEST);
     const request = BuildIssueRequest(tokens);
     // Construct info for xhr signing request
-    let xhrInfo = {newUrl: reqUrl, requestBody: `blinded-tokens=${request}&captcha-bypass=true`, tokens: tokens};
+    const xhrInfo = {newUrl: reqUrl, requestBody: `blinded-tokens=${request}&captcha-bypass=true`, tokens: tokens};
     return xhrInfo;
 }
 
 /**
  * Sends an XHR request containing a BlindTokenRequest for signing a set of tokens
- * @param {object} xhrInfo Object containing information for the XHR that will
- * be returned.
+ * @param {Object} xhrInfo XHR info for asynchronous issuance request
  * @param {URL} url URL object
- * @param {int} tabId Tab ID for the current request
- * @returns {XMLHttpRequest}
+ * @param {Number} tabId Tab ID for the current request
+ * @return {Object}
```

Comment:
XMLHttpRequest seems more accurate than Object, did the linter complain about the former?